cibuildwheel/architecture.py

```diff
@@ -93,7 +82,7 @@ def parse_config(config: str, platform: PlatformName) -> set[Architecture]:
         return result
 
     @staticmethod
-    def native_arch(platform: PlatformName) -> Architecture | None:
+    def native_arch(platform: PlatformName) -> "Architecture | None":
```

Comment:
FYI, I tried `Self` on these (with `@classmethod`), but typing didn't seem to understand that enum is `final` (even if you tell it it's `final`), so it sees `cls.wasm32` as `Architecture.wasm32` and therefore not `Self`. Wrapping it in an extra `cls()` does make it happy, but `cls(cls.wasm32)` just to make the type checker happy wasn't worth it. Just added strings here instead of `Self`. It's technically correct since enums are always `final`.